multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java

```diff
@@ -71,7 +73,8 @@ protected void notifyInternal(DelegateExecution execution) {
 
     private void publishDynatraceEvent(DelegateExecution execution, ProcessType processType) {
         DynatraceProcessEvent startEvent = ImmutableDynatraceProcessEvent.builder()
-                                                                         .processId(VariableHandling.get(execution, Variables.CORRELATION_ID))
+                                                                         .processId(VariableHandling.get(execution,
```

Comment:
Extract correlation id into variable and use it at lines 76 and 106

Reply:
sure